src/net/account/account_client.py

```diff
@@ -89,3 +109,24 @@ async def add_transaction(
                 signature=[r, s],
             )
         )
+
+    @staticmethod
+    async def create_account(net: NetType, pk: Optional[int] = None) -> "Client":
```

Comment:
This creates `AccountClient`, not just `Client`. It is important because if user creates account without providing private key then at the end user might want to get generated private key from `AccountClient`.